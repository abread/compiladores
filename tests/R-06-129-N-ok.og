public int og() {
  auto d = 61.0;
  write d;
  d = 62.0;
  write d;
  return 0;
}

