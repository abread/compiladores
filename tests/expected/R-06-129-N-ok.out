6.1E16.2E1
